core/src/main/scala/cromwell/core/callcaching/CallCachingEligibility.scala

```diff
@@ -0,0 +1,5 @@
+package cromwell.core.callcaching
+
+sealed trait CallCachingEligibility
+case object CallCachingEligible extends CallCachingEligibility
+case class CallCachingIneligible(reason: String) extends CallCachingEligibility
```

Comment:
Generally when I say something like that, this is what I mean :) What are you looking for?